content/en/_index.md

```diff
@@ -12,8 +12,8 @@ description: A framework to secure the integrity of software supply chains
 {.display-6}
 
 <a class="btn btn-lg btn-primary me-3" href="docs/what-is-in-toto/">Learn More</a>
-<a class="btn btn-lg btn-primary me-3" href="https://github.com/in-toto/demo">Try the demo</a>
-<a class="btn btn-lg btn-primary" href="https://github.com/in-toto/friends">Explore integrations</a>
+<a class="btn btn-lg btn-primary me-3" href="docs/demo">Try the demo</a>
+<a class="btn btn-lg btn-primary" href="ecosystem/">Explore integrations</a>
```

Comment:
A link to the "Ecosystem" is in the top nav. We don't need a CTA button for this IMHO.

```suggestion
```